src/Home.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  ScrollView,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  Platform,
+  SafeAreaView,
+} from 'react-native';
+import styles from './styles';
+
+import DaySelector from './components/DaySelector';
```

Comment:
@BinaHovav can you please create a file called `index.ts` in the folder `componetnwhich contains the following:
```
export * from './DaySelector';
export * from './Meals
```
and do the same in the `models` folder?
then when you import these u can do:

```suggestion
import DaySelector, Meals from './components';
```

Reply:
i tried doing it but it didn't like it